view/rooms/new/app.js

```diff
@@ -47,7 +47,7 @@ function getPathVariables(addr){
 
   const paramType = qdata.roomType || '';
   const paramActive = qdata.active !== 'false';
-  const paramExternalId = qdata.externalId || '';
+  const paramExternalId = qdata.externalId || null;
```

Comment:
Is there any expectation or new effect (test) from '' to null?

Reply:
Thanks @cassiodias for bringing my attention to this. The reason we need to change the default from '' to null is that '' does not pass the regex validation defined for externalId